app/models/participant_profile.rb

```diff
@@ -49,6 +49,8 @@ class ParticipantProfile < ApplicationRecord
 
   self.ignored_columns = %w[user_id]
 
+  START_TERM_OPTIONS = %w[spring_2022 summer_2022].freeze
```

Comment:
This should not live in the model - this is the collection of **current** possibilities for this field and as such is a concern of the form rather than the model. Model has potentially infinite amount of start_term_options

Reply:
Agreed. I agree that those values probably shouldn’t be on the the profile, but if you do want to do that and return to improving the approach, put them on `ParticipantProfile::ECF` as they are not relevant to NPQ.

Reply:
Correct me if I'm wrong, but these are also the values for mentors?